.prettierignore

```diff
@@ -1,6 +1,10 @@
 .config
+
 dist
+docs
```

Comment:
You don't need to ignore docs when using prettier, I personally think it's a valuable tool to ensure consistent readability and clarify some parsing ambiguities that exist in Markdown.